Update node backend architecture and connect with ui events using node api

geekysuman opened the pull request:
Create node backend with MVC architecture. Connect basic ui events with node api.. some are remaining .. 
@anuragsinghbisht , @inovizz please review this..

anuragsinghbisht commented on app/actions/libraryActions.js:
```diff
@@ -94,25 +97,15 @@ export const getMyBooks = () => {
-        gas: 600000
-      }
-    ).then((response) => {
-      if(isSuccess(response)) {
```
Remove the isSuccess function also, as it's no longer needed in this file.

anuragsinghbisht commented on server/controllers/auth.controller.js:
```diff
@@ -0,0 +1,26 @@
+
+export class AuthController {
+    createAccount (req, res) {
+        console.log("calling createaccount - Nodejs");
```
Instead of using console.log statement, we can use morgan to log the output for the api's in dev environment.
https://github.com/expressjs/morgan

anuragsinghbisht commented on server/controllers/auth.controller.js:
```diff
@@ -0,0 +1,26 @@
+    createAccount (req, res) {
+        console.log("calling createaccount - Nodejs");
+        request({
+            url: 'http://localhost:8545',
```
Instead of hardcoding the url 'http://localhost:8545', it will be better if we use the config file to store the url parameters.

geekysuman commented:
Updated.. 
Added **Morgan** for console and added **'http://localhost:8545'** url in config file.
There are some functions which using **isSuccess** method, once I update those then I'll remove this method.
please review once @anuragsinghbisht and @inovizz

anuragsinghbisht commented:
+1

nrchandan commented:
Please restrict the length of the commit title to 50 chars. And provide detail in the message body.
Hint: Don't use "commit -m".
